orchestrators/Orchestrator.py

```diff
@@ -7,19 +7,19 @@
 from dotenv import load_dotenv
 
 class Orchestrator(ABC):
-    load_dotenv()
-
-    DEBUG = os.environ.get("DEBUG", "false")
-    DEBUG_LOGGING = DEBUG.lower() == "true"
-
     @abstractmethod
-    def conversation_with_data(self, request_body):
+    def conversation_with_data(self, request_body, message_uuid):
         pass
 
     @abstractmethod
-    def conversation_without_data(self, request_body):
+    def conversation_without_data(self, request_body, message_uuid):
         pass
 
+    load_dotenv()
```

Comment:
What was the reason for moving this (and the logging stuff below) to after the abstract methods? The original location at the top of the class def seems more natural.

Reply:
I just put that back to where it was before.